src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -200,30 +249,29 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
           <VulnerabilitiesBlock scores={ccviScores} region={region} />
         </LocationPageBlock>
         <LocationPageBlock>
-          <Recommend
-            introCopy={recommendationsIntro}
-            recommendations={recommendationsMainContent}
-            locationName={region.fullName}
-            shareUrl={recommendsShareUrl}
-            shareQuote={recommendsShareQuote}
-            recommendationsRef={recommendationsRef}
-            feedbackFormUrl={recommendationsFeedbackForm}
-            fedLevel={getFedLevel(projections.primary)}
-            harvardLevel={getHarvardLevel(projections.primary)}
-            harvardModalLocationCopy={recommendationsHarvardModalCopy}
-            fedModalLocationCopy={recommendationsFedModalCopy}
-          />
+          {!projections ? (
```

Comment:
and now each other thing displaying is conditional on projections being available.